TemporaryOysters/bangazon_website
# added login, register, and logout methods

##Pull Requests



##Descriptions:
Added methods for login, register, and logout.  

* Please describe what issue this PR solves











##Steps to Test:


-git fetch
-Checkout the th-logins branch
-Delete your own db.sqlite3 file
-From terminal, within your main bangazon_website folder, run python bangazon_website/manage.py makemigrations bangazon_webstore
-From same folder run python bangazon_website/manage.py migrate
-From the same folder run python bangazon_website/manage.py test bangazon_webstore
-You should have 16 tests passing

##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?


* Any command line utilities to run?
